tensorflow/python/kernel_tests/decode_raw_op_test.py

```diff
@@ -20,14 +20,17 @@
 from __future__ import print_function
 
 import tensorflow as tf
-
+import sys
 
 class DecodeRawOpTest(tf.test.TestCase):
 
   def testToUint8(self):
     with self.test_session():
       in_bytes = tf.placeholder(tf.string, shape=[2])
-      decode = tf.decode_raw(in_bytes, out_type=tf.uint8)
+      if sys.byteorder == "big":
+        decode = tf.decode_raw(in_bytes, out_type=tf.uint8, little_endian=False)
```

Comment:
decode_raw's implementation in decode_raw_op.cc  has an assert that the little_endian_ flag is kLittleEndian (or the sizeof is 1). You probably need to implement endian swapping in that function (and remove the assert)